src/test/java/seedu/address/model/unique/UniqueListTest.java

```diff
@@ -175,16 +150,6 @@ public void sets_nullList_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> uniqueUniqueMockList.setList((List<UniqueMock>) null));
     }
 
-    @Test
-    public void sets_list_replacesOwnListWithProvidedList() {
-        uniqueUniqueMockList.add(u1);
-        List<UniqueMock> personList = Collections.singletonList(u3);
-        uniqueUniqueMockList.setList(personList);
-        UniqueList<UniqueMock> expectedUniqueUniqueMockList = new UniqueList<UniqueMock>();
-        expectedUniqueUniqueMockList.add(u3);
-        assertEquals(expectedUniqueUniqueMockList, uniqueUniqueMockList);
-    }
-
     @Test
     public void sets_listWithDuplicateUniqueMocks_throwsDuplicateUniqueMockException() {
         List<UniqueMock> listWithDuplicateUniqueMocks = Arrays.asList(u1, u1);
```

Comment:
This entire file, should not have test cases being deleted. I forgot to name to change all references to person